src/handle_action.cpp

```diff
@@ -674,6 +674,25 @@ static void smash()
             crit->use_mech_power( -3 );
         }
     }
+    for( std::pair<const field_type_id, field_entry> &fd_to_smsh : m.field_at( smashp ) ) {
+        const map_bash_info &bash_info = fd_to_smsh.first->bash_info;
+        if( bash_info.str_min == -1 ) {
+            continue;
+        }
+        if( smashskill < bash_info.str_min && one_in( 10 ) ) {
+            // %s is the smashed field
```

Comment:
If this is intended to be for the translators, it must be 
```suggestion
            //~ %s is the smashed field
```